README.md

```diff
@@ -43,16 +43,21 @@ Import and run the `setup` function in your `test-helper.js` file:
 
 ```js
 // tests/test-helper.js
+import * as QUnit from 'qunit';
```

Comment:
I wasn't sure if it'd be good to show the default import statements,

```javascript
import Application from 'my-project-name/app';
import config from 'my-project-name/config/environment';
import { setApplication } from '@ember/test-helpers';
import { start } from 'ember-qunit';
```

to make it even clearer where we get `setApplication`, `Application`, `config`, and `start` from.

I can push another commit to add these import statements if you'd like.